devdoc/channel_2.md

```diff
@@ -0,0 +1,72 @@
+# channel_2 requirements:
+
+## Overview
+
+The current implementation of `channel` has certain pitfalls. `channel_2` aims to address these pitfalls by making the following changes:
```

Comment:
>certain pitfalls. `channel_2` aims to address these pitfalls by making the following changes: [](http://example.com/codeflow?start=44&length=93)

Would be good to mention what the pitfalls are. Otherwise it sounds like this:

"There were some problems with channel. We're not going to say what the problems are but here's a solution to the unknown problems" :-)